### PR TITLE
fix(content): Offline mode

### LINKED ISSUE #229: Content And Offline mode
Hello,

I'm trying to build an web app using this module and the [pwa module](https://pwa.nuxtjs.org/). When I try to go offline, the navigator says that he can't fetch the database. Is this a bug, or I miss something or is this impossible ?

image

### PR DESCRIPTION
**Pull request by Barbapapazes**
Fix issue when using app in SPA mode. The DB wasn't call correctly (using a `Date.now()` which makes unusable offline mode)

## Types of changes

- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

It resolve #229 ! Using a env in the runtimeConfig and passing it to the static plugin, the value is the same during the app is alive but change when there is a new build !

Maybe I can use a key generation but don't kwon how !

## Checklist:
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes (if not applicable, please state why)

### CODE REVIEW

**Comment by benjamincanac** [comment]
@Barbapapazes Really nice PR! Just need to fix the tests and it will be good to go 🙂

**Comment by Barbapapazes** [comment]
Tests is ok !

**Review by benjamincanac — APPROVED** [review]

File: packages/content/lib/index.js [hunk 1]
```diff
@@ -138,12 +156,20 @@ module.exports = async function (moduleOptions) {
 
   /* istanbul ignore if */
   if (isSSG) {
+    // Create a hash to fetch the database
+    const dbHash = hash(JSON.stringify(database.items._data)).substr(0, 8)
```
Comment (pi0) [inline packages/content/lib/index.js:160]: Is it okay that not other database changes invalidate hash?

File: packages/content/lib/index.js [hunk 2]
```diff
@@ -138,12 +156,20 @@ module.exports = async function (moduleOptions) {
...
+    // Pass the hash to the publicRuntimeConfig to be used in client side
+    this.options.publicRuntimeConfig._CONTENT_HASH_DB = dbHash
```
Comment (pi0) [inline packages/content/lib/index.js:162]: runtimeConfig is a js object not env to be uppercase. We may use `.publicRuntimeConfig.content = { dbHash }`

**Review by pi0 — COMMENTED** [review]
Nice improvement!

File: packages/content/lib/index.js (hunk 2 above)
Comment (Atinux) [reply packages/content/lib/index.js:162]: Agree :)

File: packages/content/lib/index.js (hunk 1 above)
Comment (Atinux) [reply packages/content/lib/index.js:160]: Yes since it's only stored in items, right @benjamincanac ?

Comment (Barbapapazes) [reply packages/content/lib/index.js:160]: I can use the entire database file to generate the hash if you think it's better but I analyse the file and only the field items._data contains the data !

File: packages/content/lib/index.js (hunk 2 above)
Comment (Barbapapazes) [reply packages/content/lib/index.js:162]: ok, so I change this !

**Review by Atinux — APPROVED** [review]
Awesome work 🔥

**Pull request merged**